src/bindings/python/src/openvino/preprocess/README.md

```diff
@@ -0,0 +1,60 @@
+# Torchvision to OpenVINO preprocessing converter
+
+Torchvision to OpenVINO preprocessing converter allows you to use an existing `torchvision.transforms` object to automatically translate it to OpenVINO preprocessing. It is then being embedded into the model, resulting in better inference performance.
+
+
+## Supported transforms
+
+Current torchvision to OpenVINO preprocessing converter does not support all torchvision transforms.
+
+Supported operations:
+- `transforms.Compose`
+- `transforms.Normalize`
+- `transforms.ConvertImageDtype`
+- `transforms.Grayscale`
+- `transforms.Pad`
+- `transforms.ToTensor`
+- `transforms.CenterCrop`
+- `transforms.Resize`
+
+## Example usage
+
+```python
+preprocess_pipeline = torchvision.transforms.Compose(
+    [
+        torchvision.transforms.Resize(256, interpolation=transforms.InterpolationMode.NEAREST),
```

Comment:
Do we support Bilinear interpolation from Pillow library? I noticed many times that it is a default one.

Reply:
I did some digging and it turned out that while we have Pillow interpolation modes implemented in C++, we do not have Python bindings for them. Let's continue this topic in ticket 124943 and its Good First Issue https://github.com/openvinotoolkit/openvino/issues/21042.